Sources/TuistAutomation/Simulator/SimulatorRuntime.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+import TSCBasic
+
+/// It represents a runtime that is available in the system. The list of available runtimes is obtained
+/// using Xcode's simctl cli tool.
+struct SimulatorRuntime: Decodable, Hashable, CustomStringConvertible {
+    /// Runtime bundle path (e.g. /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime)
+    let bundlePath: AbsolutePath
+
+    /// Runtime build version (e.g. 17F61)
+    let buildVersion: String
+
+    /// Runtime root (e.g. /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes\/iOS.simruntime\Contents/Resources/RuntimeRoot)
+    let runtimeRoot: AbsolutePath
+
+    /// Runtime identifier (e.g. com.apple.CoreSimulator.SimRuntime.iOS-13-5)
+    let identifier: String
+
+    /// Runtime version (e.g. 13.5)
+    let version: String
+
+    // True if the runtime is available.
+    let isAvailable: Bool
+
+    // Name of the runtime (e.g. iOS 13.5)
+    let name: String
+
+    init(bundlePath: AbsolutePath,
+         buildVersion: String,
+         runtimeRoot: AbsolutePath,
+         identifier: String,
+         version: String,
+         isAvailable: Bool,
+         name: String) {
+        self.bundlePath = bundlePath
+        self.buildVersion = buildVersion
+        self.runtimeRoot = runtimeRoot
+        self.identifier = identifier
+        self.version = version
+        self.isAvailable = isAvailable
+        self.name = name
+    }
```

Comment:
Since this is a struct - the init should automatically be synthesised for you!

Reply:
It's synthesized internally but not publicly 😬 _(unless Apple has changed it)_

Reply:
internal synthesis is default, if you meant this to be public then I would add that to the signature ;p